airbyte-webapp/src/components/base/Button/types.tsx

```diff
@@ -1,10 +1,18 @@
+import React, { MutableRefObject } from "react";
+
+type ButtonSize = "xs" | "sm" | "lg";
+type ButtonVariant = "primary" | "secondary" | "danger" | "light";
+
 export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  buttonRef?: MutableRefObject<HTMLButtonElement | null>;
```

Comment:
`forwardRef` could be used to forward the button ref instead.

Reply:
fixed